keylime/cloud_verifier_tornado.py

```diff
@@ -421,6 +422,21 @@ def post(self):
                     logger.warning("POST returning 400 response. Expected non zero content length.")
                 else:
                     json_body = json.loads(self.request.body)
+
+                    # Handle allowlists
+                    allowlist_bundle = json.loads(json_body.get("ima_policy"))
```

Comment:
This is an API change, but because it is only the backbend API incrementing the minor version is fine.